internal/oidc/auth/auth_handler_test.go

```diff
@@ -913,3 +1331,7 @@ func requireEqualURLs(t *testing.T, actualURL string, expectedURL string, ignore
 	}
 	require.Equal(t, expectedLocationQuery, actualLocationQuery)
 }
+
+func stringPtr(s string) *string {
```

Comment:
`k8s.io/utils/pointer.StringPtr`

Reply:
Replaced across our entire code base in f0652c1ce161472accae9f3ac8a46d88a65fd94d